### ISSUE TITLE
Mi Home 360 camera (chuabgmi.camera.ipc019) stopped working

### ISSUE THREAD

**Issue by new-kirte** [al-one/hass-xiaomi-miot#976]
All of a sudden camera stopped working and I am getting the error:
2023-01-04 18:08:19.019 ERROR (MainThread) [custom_components.xiaomi_miot.camera] Play Area Camera(chuangmi.camera.ipc019): Got MiioException while fetching the state: Unable to discover the device 192.168.100.133, mapping: {'camera_control.on': {'siid': 2, 'piid': 1}, 'camera_control.image_rollover': {'siid': 2, 'piid': 2}, 'camera_control.night_shot': {'siid': 2, 'piid': 3}, 'camera_control.time_watermark': {'siid': 2, 'piid': 4}, 'camera_control.wdr_mode': {'siid': 2, 'piid': 5}, 'glimmer_full_color-2-6': {'siid': 2, 'piid': 6}, 'camera_control.recording_mode': {'siid': 2, 'piid': 7}, 'camera_control.motion_tracking': {'siid': 2, 'piid': 8}, 'stream_status-3-1': {'siid': 3, 'piid': 1}}, max_properties: 10/9

**Comment by new-kirte** [comment]
It seems that component setting "automatic" diverts to local, so the camera needs miot_cloud: true in customize.xml

**Comment by al-one** [comment]
Yes, the device is integrated via LAN by default. If it is not in the same network as HA, you need to enable the `miot_cloud` through customization.

**Comment by new-kirte** [comment]
> Yes, the device is integrated via LAN by default. If it is not in the same network as HA, you need to enable the `miot_cloud` through customization.

Thanks, but then what is the difference between Automatic and Local in the configuration? Automatic was earning for almost 12 months and broke in the last version.

**Comment by al-one** [comment]
- `Automatic mode` will only integrate known Wi-Fi devices that support integrated in the LAN locally, and other devices will be integrated via cloud.
  - [`xiaomi_miot/core/miot_local_devices.py`](https://github.com/al-one/hass-xiaomi-miot/blob/master/custom_components/xiaomi_miot/core/miot_local_devices.py)
  - [`xiaomi_miot/core/miio2miot_specs.py`](https://github.com/al-one/hass-xiaomi-miot/blob/master/custom_components/xiaomi_miot/core/miio2miot_specs.py)
- `Local mode` will integrate all included devices locally, which means that if you include devices that do not support local (such as: ble, zigbee), they will not be available.
